### PR TITLE
feat(shields): Add swept keyboard

### PR DESCRIPTION
**Pull request by equiman**
I made this PR using a `swept` keyboard and tested this new shield using a private repository.

front

PCB and components are available here as Open Source:
https://github.com/AYM1607/swept-crkbd/

## Board/Shield Check-list
 - [X] This board/shield is tested working on real hardware
 - [X] Definitions follow the general style of other shields/boards upstream ([Reference](https://zmk.dev/docs/development/new-shield))
 - [X] `.zmk.yml` metadata file added
 - [X] Proper Copyright + License headers added to applicable files (Generally, we stick to "The ZMK Contributors" for copyrights to help avoid churn when files get edited)
 - [X] General consistent formatting of DeviceTree files
 - [X] Keymaps do not use deprecated key defines (Check using the [upgrader tool](https://zmk.dev/docs/codes/keymap-upgrader))
 - [X] `&pro_micro` used in favor of `&pro_micro_d/a` if applicable
 - [X] If split, no name added for the right/peripheral half
 - [X] Kconfig.defconfig file correctly wraps *all* configuration in conditional on the shield symbol
 - [X] `.conf` file has optional extra features commented out
 - [X] Keyboard/PCB is part of a shipped group buy or is generally available in stock to purchase (OSH/personal projects without general availability should create a zmk-config repo instead)

### CODE REVIEW

File: app/boards/shields/swept/Kconfig.shield [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2020 Pete Johanson
```
Comment (caksoylar) [inline app/boards/shields/swept/Kconfig.shield:1]: Need to update the copyright headers to `2023 The ZMK Contributors` across the files.

File: app/boards/shields/swept/boards/nice_nano.overlay [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#include <dt-bindings/led/led.h>
```
Comment (caksoylar) [inline app/boards/shields/swept/boards/nice_nano.overlay:1]: I'd also consider adding an overlay for other common boards, at least for `nice_nano_v2` until a better solution to #885 is implemented.

Comment (equiman) [reply app/boards/shields/swept/boards/nice_nano.overlay:1]: Actually, this board only supports v2, so I'll change it. Thanks

File: app/boards/shields/swept/Kconfig.shield (hunk 1 above)
Comment (equiman) [reply app/boards/shields/swept/Kconfig.shield:1]: Ok. I'll fix that.

**Comment by Nicell** [comment]
I noticed the repo for this shield says to use the corne firmware in ZMK. Is there any notable difference between this firmware and the corne firmware?

**Comment by equiman** [comment]
Yeah, it was using corne because is the shield with the nearest behavior available on ZMK. 

Swept it a more minimalistic keyboard and for example, doesn't have or need support for LEDs. 

Also, I've added this shield because some users are confused while pairing their keyboards, expecting to see 'Swept' instead of 'Corne'

**Comment by equiman** [comment]
stop working with Zephyr 3.2. I've decided rollout changes and use corne that is officially supported and maintained.

**Pull request closed without merging**